lib/rest-api.php

```diff
@@ -307,6 +307,40 @@ function gutenberg_register_taxonomy_prepare_functions( $taxonomy ) {
 }
 add_filter( 'registered_taxonomy', 'gutenberg_register_taxonomy_prepare_functions' );
 
+/**
+ * Ensure that the wp-json index contains the 'theme-supports' setting as
+ * part of its site info elements.
+ *
+ * @see https://core.trac.wordpress.org/ticket/45016
+ *
+ * @param WP_REST_Response $response WP REST API response of the wp-json index.
+ * @return WP_REST_Response Response that contains theme-supports.
+ */
+function gutenberg_ensure_wp_json_has_theme_supports( $response ) {
```

Comment:
This `gutenberg_ensure_wp_json_has_theme_supports()` function needs to be removed from the pull request because it no longer exists in `master`.

Reply:
Whoa, sorry about that! I must have changed things _just_ enough for the merge of master to keep them instead of removing them. Changes to `lib/rest-api.php` have been reverted.